lib/matplotlib/text.py

```diff
@@ -1796,10 +1787,22 @@ def contains(self, event):
             contains = contains or in_patch
         return contains, tinfo
 
+    @property
+    def xycoords(self):
+        return self._xycoords
+
+    @xycoords.setter
+    def xycoords(self, xycoords):
+        def is_offset(s): return isinstance(s, str) and s.startswith("offset")
```

Comment:
```suggestion
        def is_offset(s):
            return isinstance(s, str) and s.startswith("offset")
            
```
Let's give the code space  to breathe.

Reply:
done